main.go

```diff
@@ -63,6 +63,11 @@ func main() {
 			Usage:  "kubernetes auth `TOKEN`",
 			EnvVar: "KUBE_TOKEN,PLUGIN_KUBE_TOKEN",
 		},
+		cli.StringFlag{
+			Name:   "config",
+			Usage:  "Env file location",
+			EnvVar: "CONFIG_FILE",
```

Comment:
for consistency and use in a drone plugin can we plus PLUGIN_CONFIG_FILE in the EnvVar